<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_iloscOdcinkow">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>230</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ilość odcnków blokowych (SBL):</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_wybierzBlokade">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Wybierz rodzaj blokady</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blokada półsamoczynna (PBL)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dwustawna blokada samoczynna (SBL)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Trzystawna blokada samoczynna (SBL)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Czterostawna blokada samoczynna (SBL)</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spb_dlugoscLinii">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>169</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wybierzBlokade">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz blokadę:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_dlugoscOdcinkow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="sce_scene">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>1181</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_predkoscSymulacji">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_dlugoscSkladu">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>600</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_maksymalnaPredkosc">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>139</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_predkoscSymulacji">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prędkość symulacji (max. x10): </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_maksymalnaPredkosc">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>140</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maksymalna prędkość składu [km/h]:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dlugoscLinii">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Długość linii [km] (PBL):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spb_iloscOdcinkow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>[]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dlugoscOdcinkow">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>200</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Długość odcnków blokowych [m] (SBL):</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_dlugoscSkladu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Długość składu [m]:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddTrain">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj drugi pociąg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd1Train">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj pociąg</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>[]</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbFirst">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbFifth">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Blinking</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbSecond">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>240</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Blinking</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbThird">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>260</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chbFirst">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Blinking</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbSecond">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbFifth">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbFourth">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>280</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyczyść</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPause">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_PBLSignals">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Wybierz sygnał</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S10a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S11a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S12a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S13a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sygnał zastępczy</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
